packages/amplify-migration-tests/src/configure/index.ts

```diff
@@ -0,0 +1,51 @@
+import * as nexpect from 'nexpect';
+
+import { getCLIPath, isCI } from '../utils';
+type AmplifyConfiguration = {
+  accessKeyId: string;
+  secretAccessKey: string;
+  profileName?: string;
+};
+
+const defaultSettings = {
+  profileName: 'amplify-integ-test-user',
+  region: '\r',
+  userName: '\r',
+};
+
+const MANDATORY_PARAMS = ['accessKeyId', 'secretAccessKey'];
+export default function amplifyConfigure(settings: AmplifyConfiguration, verbose: Boolean = isCI() ? false : true) {
```

Comment:
This function can be move to e2e core as the same thing is done in amplify-e2e-tests. Could you move this to e2e-core and use it in both places. Also we are not using verbose